src/main/java/medicaldoctor/entities/UserType.java

```diff
@@ -104,5 +105,25 @@ public boolean equals(Object obj) {
         }
         return true;
     }
+    
+    public String getDashboardName(){
```

Comment:
better, but i meant as an actual property on the object, like adding a new column to the entity.
 this still has if statements :)